conans/model/build_info.py

```diff
@@ -34,8 +34,10 @@ def __init__(self):
         self.cxxflags = []  # C++ compilation flags
         self.sharedlinkflags = []  # linker flags
         self.exelinkflags = []  # linker flags
+        self.build_modules = []
```

Comment:
It feels a bit strange to have paths inside this variable. Following the current pattern in`cpp_info` this should be a list of module names and the path should be adjusted with `cpp_info.builddirs`

Reply:
But I think it makes sense as you need to know where the modules files are exactly located, so you need the exact path

Reply:
We discussed this and we want to simplify, otherwise you would need to start finding the files in the N Possible dirs.